MVP/Resources/Translations.no.resx

```diff
@@ -0,0 +1,456 @@
+    </data>
+    <data name="field_contributions" xml:space="preserve">
+        <value>Bidrag</value>
+    </data>
+    <data name="field_meetings" xml:space="preserve">
+        <value>Møter</value>
+    </data>
+    <data name="field_members" xml:space="preserve">
+        <value>Medlemmer</value>
+    </data>
+    <data name="field_number_of_attendees" xml:space="preserve">
+        <value>Antall Deltakere</value>
+    </data>
+    <data name="field_annual_quantity" xml:space="preserve">
+        <value>Årlig Mengde</value>
+    </data>
+    <data name="field_annual_reach" xml:space="preserve">
+        <value>Årlig Rekkevidde</value>
+    </data>
+    <data name="field_number_of_events_participated" xml:space="preserve">
+        <value>Antall Arrangementer bidratt</value>
+    </data>
+    <data name="field_number_of_feedbacks_provided" xml:space="preserve">
+        <value>Antall Tilbakemeldinger</value>
+    </data>
+    <data name="field_number_of_samples" xml:space="preserve">
+        <value>Antall Prøver</value>
+    </data>
+    <data name="field_number_of_downloads" xml:space="preserve">
+        <value>Antall Nedlastinger</value>
+    </data>
+    <data name="field_number_of_sites" xml:space="preserve">
+        <value>Antall Nettsteder</value>
+    </data>
+    <data name="field_number_of_talks" xml:space="preserve">
+        <value>Antall Foredrag</value>
+    </data>
+    <data name="field_attendees_of_talks" xml:space="preserve">
+        <value>Deltakere i Foredrag</value>
+    </data>
+    <data name="field_number_of_followers" xml:space="preserve">
+        <value>Antal Følgere</value>
+    </data>
+    <data name="field_number_of_videos" xml:space="preserve">
+        <value>Antall Videoer</value>
+    </data>
+    <data name="field_number_of_events" xml:space="preserve">
+        <value>Antall Arrangementer</value>
+    </data>
+    <data name="onboarding_1_title" xml:space="preserve">
+        <value>Enkelt og effektiv</value>
+    </data>
+    <data name="onboarding_1_description" xml:space="preserve">
+        <value>Administrer alle samfunnsaktivitetene dine fra håndflaten.</value>
+    </data>
+    <data name="onboarding_3_title" xml:space="preserve">
+        <value>Rask Tilsetning</value>
+    </data>
+    <data name="onboarding_3_description" xml:space="preserve">
+        <value>Lag direkte bidrag basert på en URL på klippetavla. Fyller ut alt mulig for deg, slik at du raskt kan legge til nye bidrag.</value>
+    </data>
+    <data name="acknowledgementtype_translator" xml:space="preserve">
+        <value>Oversetter</value>
+    </data>
+    <data name="acknowledgementtype_tester" xml:space="preserve">
+        <value>Tester</value>
+    </data>
+    <data name="about_credits_description" xml:space="preserve">
+        <value>Uten det råeste folket som er oppført nedenfor, denne appen kunne ikke ha vært mulig. Disse har derfor fortjent en plass her å feire deres bidrag.</value>
```

Comment:
Everywhere I look råeste seems to translate to raw? :D

Reply:
Yes, literally that is what it means. However, it's used in speaking language as 'cool' or 'awesome'